cmd/vela-worker/exec.go

```diff
@@ -268,6 +268,14 @@ func (w *Worker) exec(index int, config *library.Worker) error {
 		return nil
 	}
 
+	logger.Info("assembling build")
+	// assemble the build with the executor
+	err = _executor.AssembleBuild(timeoutCtx)
```

Comment:
whats the reason for moving Assemble below Stream?

Reply:
I would say that Easton could probably add more to this, but when testing this it kept waiting 5 minutes before it would actually run the build and swapping this code around fixed that.

Reply:
hm interesting... would that be a separate issue? or is it related to the privileged images logging?

Reply:
I made it a separate issue!